demo-is/config-volumes/identity-server/deployment.toml

```diff
@@ -126,3 +142,11 @@ EmailOTPEnableByUserClaim = true
 CaptureAndUpdateEmailAddress = true
 showEmailAddressInUI = true
 useEventHandlerBasedEmailSender = true
+
+#Enable access for sample apps
+[identity_mgt.user_onboarding]
```

Comment:
I wish these settings would be disabled in the management portal UI when they are defined in deployment.toml.

Reply:
+1